src/http_server.rs

```diff
@@ -30,9 +68,19 @@ impl<T: Controller> HttpServer<T> {
 
         let addrs: Vec<_> = self.controller.http_as_addrs().unwrap().collect();
 
-        thread::Builder::new().name("HttpServer".to_owned())
+        debug!("Starting server with cert: {:?} and key: {:?}", cert, key);
+
+        let mut config = TlsSniConfig::new();
+        config.add_ssl_cert("foxbox.local".to_owned(), cert, key).unwrap();
+        config.add_ssl_cert("test.knilxof.org".to_owned(), self.controller.get_remote_tls_crt(), self.controller.get_remote_tls_key()).unwrap();
```

Comment:
Do we need the `add_ssl_cert` function? Could we instead pass to `init_ssl_context` the list  of hostnames, certs and keys?